CONTRIBUTING.md

```diff
@@ -20,4 +20,10 @@ again.
 All submissions, including submissions by project members, require review. We
 use GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
-information on using pull requests.
\ No newline at end of file
+information on using pull requests.
+
+## Setup: VSCode
+
+Copy the `.vscode.recommended` folder into `.vscode` and edit the contents to your liking.
```

Comment:
Let's put it in the `/.vscode` folder instead, so the developer doesn't need to do anything else. I'd like it to just work on my multiple environments.